### PR TITLE
Add tap and capacitive touch component support

### PR DESCRIPTION
**Pull request by bcjordan**
Use updated `johnny-five` with firmata browserify fix. Relies on some new component definitions referenced by branch until merged which will eventually live at https://github.com/rwaldron/playground-io/pull/2

Little demo of components in action in Code Studio Applab: https://www.dropbox.com/s/3n1sb8trwwpjyjr/makerlab-demo.screenflow.mp4?dl=0

### CODE REVIEW

File: code-studio/package.json [hunk 1]
```diff
@@ -57,15 +57,15 @@
     "factor-bundle": "^2.5.0",
     "http-server": "^0.9.0",
     "istanbul": "^0.4.2",
-    "johnny-five": "bcjordan/johnny-five#with-firmata-fix",
+    "johnny-five": "^0.9.37",
```
Comment (bcjordan) [inline code-studio/package.json:60]: This fix was merged in, so updated to npm version with fix.

File: apps/src/makerlab/BoardController.js [hunk 2]
```diff
@@ -195,8 +202,12 @@ function initializeCircuitPlaygroundComponents() {
       freq: 100
     }),
 
-    gyro: new five.Gyro({
+    accelerometer: new five.Gyro({
```
Comment (islemaster) [inline apps/src/makerlab/BoardController.js]: What's the purpose of the rename here, and/or is this not used anywhere yet?  Usually with a change like this I look for a corresponding usage change.

**Comment by islemaster** [comment]
LGTM!  Nice demo!

File: apps/src/makerlab/BoardController.js (hunk 2 above)
Comment (bcjordan) [inline apps/src/makerlab/BoardController.js]: Great question, someone noted the component itself is technically an accelerometer so wanted to be accurate. It actually looks like there's a `five.Accelerometer` I most likely will change the `PlaygroundIO` plugin to extend instead.

Re: usage, another good Q, this is not referenced in the codebase yet, though I updated a temporary usage instructions doc. The API is subject to change a good deal over the next couple of weeks.

**Comment by bcjordan** [comment]
Fail on merge: `Error Cannot find module 'mock-firmata' from '[CDO]/code-studio/node_modules/johnny-five/lib'` — investigating.

**Pull request merged**